packages/pancake-uikit/src/components/Svg/Icons/ArrowFirst.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Svg from "../Svg";
+import { SvgProps } from "../types";
+
+const Icon: React.FC<SvgProps> = (props) => {
+  return (
+    <Svg viewBox="0 0 24 25" {...props}>
```

Comment:
We need to figure this out eventually. I suspect it is Figma's export functionality but this should be 24x24.

Reply:
Think it's safe to overwrite the 25 > 24?  It's definitely 24px in Figma. Is the risk just weird scaling?